dnf-docker-test/features/steps/rpm_utils.py

```diff
@@ -120,3 +120,13 @@ def analyze_state(pre, post):
 
     # Most probably we just compare different packages
     assert False, "{!r} -> {!r}".format(pre_nevra, post_nevra)
+
+def is_installonly_pkg(pkg):
+    """
+    Returns True if the package is an installonly package, i.e. provides
+    a package from the defined set.
+    """
+    installonly_pkgs = set([u'kernel', u'kernel-PAE', u'installonlypkg(kernel)', u'installonlypkg(kernel-module)', u'installonlypkg(vm)', u'multiversion(kernel)'])
```

Comment:
I'd like to have this set moved out of the function (and thus constructed only once) and named in upper case. Now the set is freshly build in each call of the function.

Reply:
and please split this long line to not exceed 100 chars

Reply:
OK. Done.